tabs/pid_tuning.js

```diff
@@ -8,9 +8,7 @@ var SUPEREXPO_FEATURE_BIT = 23;
 
 TABS.pid_tuning.initialize = function (callback) {
     var self = this;
-    if (GUI.active_tab != 'pid_tuning') {
-        GUI.active_tab = 'pid_tuning';
-    }
+    if (GUI.active_tab != 'pid_tuning') { GUI.active_tab = 'pid_tuning'; }
 
```

Comment:
I haven't been able to find a style guide for this project. We should start one. But as a rule of thumb 'keep the style of new stuff that you are adding in line with the style that is used in the existing stuff, and try to leave everything as clean or if possible cleaner than when you found it' is a good approach in my opinion.